qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/codeactions/QuteCodeActionForUnknownProperty.java

```diff
@@ -76,6 +79,11 @@ public void doCodeActions(CodeActionRequest request, List<CompletableFuture<Void
 			if (baseResolvedType != null) {
 				doCodeActionToCreateProperty(propertyPart, baseResolvedType, template, diagnostic, resolver,
 						sharedSettings, codeActionResolveFutures, codeActions);
+
+				PropertyPart part = (PropertyPart) node;
+				String resolvedType = baseResolvedType.getSignature();

```

Comment:
Remove this variable and uses baseResolvedType to getFields. You dont  need to retrieve again the resolved type from javaCache